src/helpers/spacewalk.ts

```diff
@@ -128,6 +128,13 @@ export function currencyToString(currency: SpacewalkPrimitivesCurrencyId, tenant
   }
 }
 
+export function currencyToStellarAssetCode(currency: SpacewalkPrimitivesCurrencyId, tenant: TenantName = TenantName.Pendulum) {
+  if (!currency.isStellar) {
+    throw Error('Not an Stellar asset');
```

Comment:
This function throws an error that is not caught by any calling function of the transfer dialogs. If you test this on Amplitude, this error is thrown a lot and since it's an uncaught error, the application stops working appropriately (ie. you cannot close any dialogs or do anything else).
I recommend we wrap this in a try-catch block like above and return `'Unknown'` or `null`.

Reply:
Also, as defined [here](https://github.com/pendulum-chain/portal/blob/de0fe699d2b70882829aff272d30a909bfe1491c/src/pages/bridge/TransfersColumns.tsx#L19-L20), the `asset` is a string and not of type `SpacewalkPrimitivesCurrencyId` which is probably the reason why this function errors all the time.

Reply:
Thanks for the catch. Couple of things here:
1. `currencyToStellarAssetCode()` takes the same signature as `currencyToString()`, that's why it should receive a`SpacewalkPrimitivesCurrencyId`, and that was already working good afaik (i.e. I just replaced the uses of `currencyToString` with `currencyToStellarAssetCode`, the expected behavior was to show only the asset code, instead of the whole `code:issuer`)
2. I didn't realize that there's another existing function called `convertCurrencyToStellarAsset()` which already does the check, even though it doesn't throw an error, which was a bad decision after all. I will use that one and check why is failing so much.